### ISSUE TITLE
Missing visual and programmatic labels for form controls

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3123]
Some related form controls, e.g. the three "Multiply color" or "Rotation" spin buttons, share the same visible label which appears as left-justified above the first field. This is problematic for sighted users who may have difficulty determining the purpose of the remaining fields in the group, other than the first one. Due to the current focus styling,  keyboard users cannot see any indication of when the form controls are reached. 

Screen reader users hear a label announced only for the first input field, while the  remaining ones in the group are unlabeled, since they are programmatically associated  with a `<label>` element with no text. 

**Suggested Solution** 
Use a `<fieldset>` to group the control with the visible group label wrapped by the `<legend>`. Provide visible and programmatic labels for each <input> in the set to  clearly identify their purpose. 

Severity: 1
